pkg/tables/tables.go

```diff
@@ -2,12 +2,48 @@ package tables
 
 import (
 	"context"
+	"fmt"
 	"math/big"
 
 	"github.com/ethereum/go-ethereum/common"
-	"github.com/textileio/go-tableland/internal/tableland"
 )
 
+// TableID is the ID of a Table.
+type TableID big.Int
```

Comment:
I moved this type here from `tableland/tableland.go` to avoid an import cycle between that file and this one. It seemed arbitrary to me where it is defined, either place makes sense, so this was the simplest work around.

In retrospect, I'm seeing that it is what caused so many file changes making the PR bigger. Sorry about that.

Reply:
I think it's fine. I might have a better opinion while being hands-on with the code in the future with the new organization.

Reply:
Cool, will leave it like this for now.